pkg/aws/metadata/handler_credentials.go

```diff
@@ -39,16 +38,13 @@ func (c *credentialsHandler) Install(router *mux.Router) {
 func (c *credentialsHandler) Handle(ctx context.Context, w http.ResponseWriter, req *http.Request) (int, error) {
 	timer := prometheus.NewTimer(handlerTimer.WithLabelValues("credentials"))
 	defer timer.ObserveDuration()
-	if statsd.Enabled {
-		defer statsd.Client.NewTiming().Send("handler.credentials")
-	}
```

Comment:
Intentional to drop the "Observed latency of credentials handler" metrics?

Reply:
Nope, that wasn't intentional. :(